### PR TITLE
Keyboard name input

### PR DESCRIPTION
**Pull request by Emiluren**

### CODE REVIEW

**Comment by Emiluren** [comment]
Solves #97

File: src/client.rs [hunk 1]
```diff
@@ -262,29 +262,40 @@ pub fn main() -> Result<(), String> {
...
             for event in event_pump.poll_iter() {
                 match event {
                     Event::Quit{..} => break 'mainloop,
                     Event::KeyDown {keycode: Some(kc), ..} => {
-                        if kc == Keycode::Return || kc == Keycode::Space {
-                            color_selection = menu_state.color_selection;
-                            plane_selection = menu_state.plane_selection;
-                            break 'menuloop;
+                        match kc {
+                            Keycode::Return => {
```
Comment (TheZoq2) [inline src/client.rs:284]: Oh no, space for start game won't work anymore :(

File: src/menu.rs [hunk 2]
```diff
@@ -35,7 +33,7 @@ pub struct MenuState {
 impl MenuState {
     pub fn new() -> MenuState {
         MenuState {
-            name: String::from(whoami::username()),
+            name: String::new(),
```
Comment (TheZoq2) [inline src/menu.rs:36]: I think we should default to `whoami`, that way you get a name even if you're too lazy to write it :)

File: src/client.rs (hunk 1 above)
Comment (malcx95) [reply src/client.rs:284]: And it shouldn't.

File: src/menu.rs (hunk 2 above)
Comment (malcx95) [reply src/menu.rs:36]: Yes, I agree.

Comment (Emiluren) [reply src/menu.rs:36]: It is in client.rs now because setting it in MenuState::new wouldn't let you keep your name

File: src/client.rs [hunk 3]
```diff
@@ -262,29 +262,40 @@ pub fn main() -> Result<(), String> {
 
     let mut color_selection = 0;
     let mut plane_selection = 0;
+    let mut name = whoami::username();
```
Comment (Emiluren) [inline src/client.rs:265]: @TheZoq2 look, is here now

**Review by TheZoq2 — APPROVED** [review]

**Review by malcx95 — APPROVED** [review]

**Pull request merged**